lib/encryption.py

```
@@ -81,7 +82,21 @@ def _load_private_key():
+        return False
+    if not data.strip().endswith("-----END OPENSSH PRIVATE KEY-----"):
+        return False
 
+    try:
+        serialization.load_ssh_private_key(
+            data.encode(),
+            password=None,  # or b"passphrase" if encrypted
+            backend=default_backend()
+        )
+        return True
+    except Exception: # pylint: disable=broad-exception-caught
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)